### ISSUE TITLE
optimisation coded, but not used 

### ISSUE THREAD

**Issue by silviatoonen** [amusecode/SeBa#23]
The Makefiles of SeBa include the optimisation flag -O through CFLAGS += -O 
However CFLAGS is not used. 

My suggestion: add a flag: CXXFLAGS += -O (or -O2)

Any objections of why not to do this?

**Comment by silviatoonen** [comment]
credit to Evert!

**Comment by silviatoonen** [comment]
from some preliminary tests, using the optimisation flags speeds up SeBa by a factor 2-3.

**Comment by rieder** [comment]
I have no objections, provided the tests on the AMUSE side don’t fail on this. 
Probably they won’t. 
Btw, is this for standalone SeBa or also the version in AMUSE (which inherits some settings from config.mk)?

**Comment by silviatoonen** [comment]
It would be for both. 
It seems that this optimisation was intended to be included, but for some time it has not been the case. 

@spzwart do you remember if for some reason it was switched of? otherwise I'll switch it back on. A factor 2-3 improvement is always nice.

**Comment by spzwart** [comment]
Yes, I think that we had done problems with the convergence in teaching RLOF. but lets them out back on,  and test the code,  see what happens.
